test_conformance/gl/test_buffers.cpp

```diff
@@ -127,14 +127,9 @@ int test_buffer_kernel(cl_context context, cl_command_queue queue,
     clKernelWrapper kernel;
     clMemWrapper streams[3];
     size_t dataSize = numElements * 16 * sizeof(cl_long);
-#if !(defined(_WIN32) && defined(_MSC_VER))
-    cl_long inData[numElements * 16], outDataCL[numElements * 16],
-        outDataGL[numElements * 16];
-#else
-    cl_long *inData = (cl_long *)_malloca(dataSize);
-    cl_long *outDataCL = (cl_long *)_malloca(dataSize);
-    cl_long *outDataGL = (cl_long *)_malloca(dataSize);
-#endif
+    std::vector<cl_long> inData(dataSize), outDataCL(dataSize),
+        outDataGL(dataSize);
```

Comment:
We should update the computation of `dataSize` after this change, since we want it to be the data size in number of elements now, not in the number of bytes.  (Basically, remove `* sizeof(cl_long)`.)